<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDI::Internal::InteractorEditorWidget</class>
 <widget class="QWidget" name="DDI::Internal::InteractorEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="Utils::MiniSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="treeLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Utils::QButtonLineEdit" name="searchLine"/>
       </item>
       <item>
        <widget class="QTreeView" name="classesTreeView"/>
       </item>
       <item>
        <widget class="QListView" name="molsListView"/>
       </item>
       <item>
        <widget class="QLabel" name="sumLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reformatOldSource">
         <property name="text">
          <string>Reformat old source</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetContainer" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="interactorLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.464 rgba(255, 255, 176, 149), stop:1 rgba(255, 255, 255, 0))</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>ATC</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="deLabel"/>
           </item>
           <item row="4" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="Views::TableView" name="atcTableView"/>
             </item>
             <item>
              <widget class="Views::TableView" name="classChildrenTableView"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>De</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>En</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="enLabel"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Fr</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="frLabel"/>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="isAutoFound">
             <property name="text">
              <string>AutoFound</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabClassInfo">
          <attribute name="title">
           <string>ClassInfo</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="1">
            <widget class="QPlainTextEdit" name="classInfoDe"/>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="classInfoEn"/>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="classInfoFr"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>De</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>En</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Fr</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Other</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QListView" name="pmidListView"/>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="commentTextEdit">
             <property name="toolTip">
              <string>Comment</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QListView" name="reviewersListView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="reference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reference (default is FreeMedForms)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateCreation">
           <property name="toolTip">
            <string>Creation date</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateUpdate">
           <property name="toolTip">
            <string>Last update</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="isReviewed">
           <property name="text">
            <string>Reviewed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="isClass">
           <property name="text">
            <string>Is interacting class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="notWarnDuplicated">
           <property name="text">
            <string>Do not warn duplication</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTextBrowser" name="messages">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>255</green>
               <blue>206</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>255</green>
               <blue>206</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="toolbarLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::QButtonLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/qbuttonlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Views::TableView</class>
   <extends>QTableView</extends>
   <header location="global">listviewplugin/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::MiniSplitter</class>
   <extends>QSplitter</extends>
   <header location="global">utils/widgets/minisplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
